lib/api/apiUtils/authorization/prepareRequestContexts.js

```diff
@@ -4,9 +4,17 @@ const { config } = require('../../../Config');
 const RequestContext = policies.RequestContext;
 const requestUtils = policies.requestUtils;
 let apiMethodAfterVersionCheck;
-const apiMethodWithVersion = { objectGetACL: true, objectPutACL: true,
-    objectGet: true, objectDelete: true, objectPutTagging: true,
-    objectGetTagging: true, objectDeleteTagging: true };
+const apiMethodWithVersion = {
```

Comment:
I feel like I am seeing this diff for the third time @dora-korpar had a PR with it before cause I commented on using a Set. Is this in like 5 places or what?

Reply:
It's the same place but we both needed to add there for our PRs, once they get merged, it's gonna be only in one place.